core/src/test/scala/unit/kafka/coordinator/group/GroupCoordinatorTest.scala

```diff
@@ -75,6 +75,12 @@ class GroupCoordinatorTest extends JUnitSuite {
   private val groupInstanceId = Some("groupInstanceId")
   private val leaderInstanceId = Some("leader")
   private val followerInstanceId = Some("follower")
+  private val invalidTimestamp = "12345"
+
+  def getInvalidMemberId(groupInstanceId : Option[String]): String = {
+      groupInstanceId.get + "-" + invalidTimestamp
```

Comment:
If you are calling .get, you are assuming groupInstanceId.isDefined. In that case, this should just be a String parameter.